rust/rope/src/engine.rs

```diff
@@ -118,11 +118,18 @@ impl Engine {
         self.find_rev(rev).map(|rev_index| self.rev_content_for_index(rev_index))
     }
 
-    /// A delta that, when applied to previous head, results in the current head. Panics
+    // TODO: doesn't this totally break in the presence of Undo currently?
```

Comment:
I don't _think_ so. Certainly it works in the case where the new revision is the result of changing the undo state.